### PR TITLE
Workaround copying to clipboard for content delivered via HTTP

### PR DESCRIPTION
**Pull request by jedrz**
`navigator.clipboard` is not accessible via HTTP. As a workaround in order to copy node's JSON, a hidden element is created, then text content is written to it and finally, second copy is executed. Because of copying twice, two events get generated so we need to discard artificial one.

### CODE REVIEW

**Comment by lciolecki** [comment]
> `navigator.clipboard` is not accessible via HTTP. As a workaround in order to copy node's JSON, a hidden element is created, then text content is written to it and finally, second copy is executed. Because of copying twice, two events get generated so we need to discard artificial one.

**Comment by lciolecki** [comment]
Please improve formatting ;)

**Comment by jedrz** [comment]
@lciolecki You are right but I don't want to untabify the file because all the history would be lost.

File: ui/client/common/ClipboardUtils.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const readText = event => (event.clipboardData || window.clipboardData).getData('text')
+
+const writeText = (text, elementId) => {
```
Comment (mproch) [inline ui/client/common/ClipboardUtils.js]: maybe some comment why this is needed and reference where does it come from?

Comment (jedrz) [reply ui/client/common/ClipboardUtils.js]: Done.

**Pull request merged**